jwst/resample/gwcs_drizzle.py

```diff
@@ -34,9 +32,9 @@ def __init__(self, product, outwcs=None, single=False,
 
         wt_scl : str, optional
             How each input image should be scaled. The choices are `exptime`
-            which scales each image by its exposure time, `expsq` which scales
-            each image by the exposure time squared, or an empty string, which
-            allows each input image to be scaled individually.
+            which scales each image by its exposure time or `expsq` which scales
```

Comment:
OK, if you're fixing punctuation you need to add commas at the ends of the words immediately preceding each occurrence of "which", i.e. "The choices are `exptime`, which scales each image by its exposure time, or `expsq`, which scales each image ...".